### DISCUSSION TITLE
How to disable Nutgram's exceptions

### DISCUSSION THREAD

**Discussion by ghost** [nutgram/nutgram#272] [Q&A]
Hi

How can i disable Nutgram's `exceptions` ?

I just want to get the result of the request even if there is an error on Telegram's side

example:

```php
$bot = new Nutgram('WRONG TOKEN');
$response = $bot->getMe();
if (!$response->ok ) {
    throw new BotException('Please send correct token:'); // this message will be send to user
}
```

another example:
```php
$bot = new Nutgram('TOKEN');
$response = $bot->getChat('WRONG CHAT ID');
if (!$response->ok ) {
    throw new BotException('Please send correct chat id:');
}
if ($reresponse->result->chat->type !== 'channel') {
    throw new BotException('Only channels is supported. please send a channel chat id:'); 
}
//etc...
```
I send different messages to the user in different situations.

But when a request encounters an error, `SergiX44\Nutgram\Telegram\Exceptions\TelegramException` or `GuzzleHttp\Exception\ClientException` fatal error happens...

I don't want to use `onApiError` or `onException` handler...

I hope you understand

Please help
Thank you

**Comment by Lukasss93** [comment]
Try this:
```php
try{
    $response = $bot->getMe();
} catch (TelegramException) {
    $bot->sendMessage('Please send correct token:');
}
```

**Reply by ghost** [reply]
It works, but every time in order not to continue the code, I have to give the exit command, which is not very interesting
```php
  try{
      $response = $bot->getMe();
  } catch (TelegramException) {
      $bot->sendMessage('Please send correct token:');

      exit; // This is not good for preventing continue code execution
  }

// some code here....
```

Can you please just add a feature that we can get the output of all sent requests?

```php
$response = $bot->getMe();
$response->result-> // .....
```

**Reply by Lukasss93** [reply]
> It works, but every time in order not to continue the code, I have to give the exit command, which is not very interesting

Never using `exit` or `die`! 😅
Just use a `return` statement 💁

> Can you please just add a feature that we can get the output of all sent requests?

No, you don't need this. It's how Nutgram works.
